docroot/themes/humsci/su_humsci_gin_admin/su_humsci_gin_admin.theme

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @file
+ * Theme functions for su_humsci_gin_admin theme.
+ */
+
+use Drupal\Component\Utility\Html;
+use Drupal\Core\Template\Attribute;
+use Drupal\paragraphs\ParagraphInterface;
+
+/**
+ * Implements hook_library_info_alter().
+ */
+function su_humsci_gin_admin_library_info_alter(&$libraries, $extension) {
+  $libraries_unset = [
+    'humsci_traditional' => 'humsci-colorful-styling',
+    'humsci_colorful' => 'humsci-traditional-styling',
+  ];
+  if ($extension === 'su_humsci_gin_admin') {
+    $theme = \Drupal::config('system.theme')->get('default');
+    unset($libraries[$libraries_unset[$theme]]);
+  }
+}
```

Comment:
Instead of unsetting them conditionally, I'd include them conditionally (opt-in instead of opt-out). I'd also use another hook to do it (see other comments for more details).

```suggestion
/**
 * Implements hook_page_attachments_alter().
 */
function su_humsci_gin_admin_page_attachments_alter(&$attachments) {
    $theme_previews = [
      'humsci_traditional',
      'humsci_colorful'
    ];
    $theme = \Drupal::config('system.theme')->get('default');
    if (in_array($theme, $theme_previews)) {
      $attachments['#attached']['library'][] = $theme . '/admin-preview';
    }
  }
```

Reply:
Done, just used a different hook. Let me know if it works or if you need me to change it.